src/middleware/anon.js

```diff
@@ -0,0 +1,41 @@
+
+function AnonymousModel (req) {
+	if (req.get('FT-Anonymous-User') === 'true') {
+		this.userIsLoggedIn = false;
+		this.userIsAnonymous = true;
+	} else {
+		this.userIsLoggedIn = true;
+		this.userIsAnonymous = false;
+	}
+}
+
+function FirstClickFreeModel () {
+	this.signInLink = '/login';
+	this.subscribeNowLink = '/product-selector?segID=400872&segmentID=676c655f-9b47-27a8-97db-ab3a6a6dbc54';
```

Comment:
shoulds this be `/products?segID=400872&segmentID=676c655f-9b47-27a8-97db-ab3a6a6dbc54` ?

Reply:
this is copied directly from n-ui. if you're sure it's wrong I can change in this PR if you like

Reply:
https://www.ft.com/product-selector returns a `404` /shrug